dev_tools/nyaml2nxdl/nyaml2nxdl_backward_tools.py

```diff
@@ -235,7 +240,7 @@ def store_root_level_comments(self, holder, comment):
     def handle_definition(self, node):
         """
         Handle definition group and its attributes
-        NOTE: Here we tried to store the order of the xml element attributes. So that we get
+        NOTE: Here we try to store the order of the xml element attributes. So that we get
```

Comment:
"attributes ~~. S~~ so that" - i.e. the sentences should be joined together.

Reply:
This is also fixed.

Reply:
L250 has not changed.

Reply:
I did not noticed  the ~~.~~ on `.`. Done